### PR TITLE
keep session alive

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add  setinterval every 60 seconds to call getUserdata method to get user information data and keep session alive in doppler

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,7 +21,13 @@ class App extends Component {
   }
 
   componentWillMount() {
-    this.getUserData();
+    this.interval = setInterval(() => {
+      this.getUserData();
```
Comment (andresmoschini) [inline src/App.js]: Should you verify if the user returned by doppler is the same that the user of the JWT token?

**Comment by fcoronelmakingsense** [comment]
> Is it possible adding any test?

I need research for test in general for react

**Comment by andresmoschini** [comment]
> > Is it possible adding any test?
> 
> I need research for test in general for react

The sooner the better.

File: src/App.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/App.js]: i added now

Comment (andresmoschini) [reply src/App.js]: What to do if it is different? we should make honor to _Doppler MVC_ response or _JWT_ token?

**Pull request merged**